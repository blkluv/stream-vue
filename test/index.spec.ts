import { mount, Wrapper } from '@vue/test-utils'
import VideoStream from '../src'

describe('stream-vue', () => {
  let wrapper: Wrapper<Vue>
  beforeEach(() => {
    wrapper = mount(VideoStream, {
      propsData: {
        src: '2938470a98fd7',
        height: 200,
        width: 500,
        controls: true,
      },
      attrs: {
        class: 'myclass',
        style: 'display: flex;',
      },
    })
  })
  it('renders HTML', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
