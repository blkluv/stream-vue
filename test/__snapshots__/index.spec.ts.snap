// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stream-vue renders HTML 1`] = `
"<div class=\\"myclass\\" style=\\"height: 200px; width: 500px;\\">
  <stream src=\\"2938470a98fd7\\" controls=\\"controls\\" currenttime=\\"0\\" volume=\\"1\\" preload=\\"none\\"></stream>
</div>"
`;
